TYPO3.Neos/Configuration/Settings.yaml

```diff
@@ -423,6 +423,9 @@ TYPO3:
       'TYPO3.Neos:Preview':
         maximumWidth: 1000
         maximumHeight: 1000
+      'TYPO3.Neos:IconPreview':
+        maximumWidth: 16
```

Comment:
Would it make sense to render these in 32x32 to have full resolution on HiDPI (retina) screens? Would work, assuming they are scaled via CSS anyway…